app/src/routes/login.tsx

```diff
@@ -72,12 +73,15 @@ function Login() {
       });
 
       onAuthStateChanged(FirebaseAuth, async (currentUser) => {
+        setIsLoading(false)
         if (currentUser) {
           const token = await getIdToken(currentUser);
           // Send id token to backend
-          axios.post('http://localhost:8080/api/auth/login',{idToken:token}, {headers:{"x-api-key":"test"}})
+          axios.post('http://ec2-34-213-210-231.us-west-2.compute.amazonaws.com/api/auth/login',{idToken:token}, {headers:{"x-api-key":"test"}})
           .then(function (response) {
-            navigate("/home",{state:response.data.session});
+            Cookies.set('sessionToken', response.data.session)
+            window.location.reload()
```

Comment:
You shouldn't need to reload and then navigate, does it break if you remove the reload call?

Reply:
i added the reload call so that when the homepage loads, all of the state loads. Without the reload, i have to manually reload it to make all the api calls go through? Is there another way of doing this?

Reply:
yeah, sorry for the late reply. I just saw the message.